src/io/dataset.cpp

```diff
@@ -1135,10 +1161,10 @@ void Dataset::ConstructHistogramsMultiVal(const data_size_t* data_indices,
 
 template <bool USE_INDICES, bool USE_HESSIAN>
 void Dataset::ConstructHistogramsInner(
-    const std::vector<int8_t>& is_feature_used, const data_size_t* data_indices,
-    data_size_t num_data, const score_t* gradients, const score_t* hessians,
-    score_t* ordered_gradients, score_t* ordered_hessians,
-    TrainingShareStates* share_state, hist_t* hist_data) const {
+  const std::vector<int8_t>& is_feature_used, const data_size_t* data_indices,
+  data_size_t num_data, const score_t* gradients, const score_t* hessians,
+  score_t* ordered_gradients, score_t* ordered_hessians,
+  TrainingShareStates* share_state, hist_t* hist_data) const {
```

Comment:
could you revert this and open a separate PR with this whitespace-only change? It's fine to change this if you want, but whitespace-only changes add unnecessary lines to this already-large PR's diff.

Reply:
Resovled via ba22deb